### PR TITLE
Fix RIA-10103 Add info on supported RAML versions and polish Readme.md

### PR DESCRIPTION
**Pull request by Alexander-Linkov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,67 +1,59 @@
-## soapui-raml-plugin
+# SoapUI RAML Plugin
 
-- Allows you to import RAML files into SoapUI for testing your REST APIs
-- Allows you to generate a REST Mock Service for a RAML file being imported
-- Allows you to udpate an existing REST Service in SoapUI from a RAML file
-- Allows you to generate a RAML file for any REST API defined in SoapUI
+This repository contains source code for the RAML plugin for ReadyAPI. Use this plugin to:
```
Comment (IlyaAvdeev) [inline README.md]: + SoapUI OS

Comment (IlyaAvdeev) [reply README.md]: + links on ReadyAPI page and SoapUI OS

File: README.md [hunk 2]
```diff
@@ -1,67 +1,59 @@
...
 
+- Import RAML files to SoapUI to test your REST APIs
+- Generate a REST Mock Service for the RAML file being imported
```
Comment (IlyaAvdeev) [inline README.md]: REST Mock/Viirtual service

File: README.md [hunk 3]
```diff
@@ -1,67 +1,59 @@
...
+- Update an existing REST Service in SoapUI from a RAML file
```
Comment (IlyaAvdeev) [inline README.md]: ReadyAPI/SoapUI

File: README.md [hunk 4]
```diff
@@ -1,67 +1,59 @@
...
+- Generate a RAML file for any REST API defined in SoapUI
```
Comment (IlyaAvdeev) [inline README.md]: and here again

File: README.md [hunk 5]
```diff
@@ -1,67 +1,59 @@
...
 
--> See the [blog-post](http://olensmar.blogspot.se/2013/12/a-raml-apihub-plugin-for-soapui.html) for a detailed overview.
+For detailed information, please visit the following page:<br/>[http://olensmar.blogspot.se/2013/12/a-raml-apihub-plugin-for-soapui.html](http://olensmar.blogspot.se/2013/12/a-raml-apihub-plugin-for-soapui.html).
```
Comment (IlyaAvdeev) [inline README.md]: we can skip this. If some documentation required point out to ReadyAPI docs

File: README.md [hunk 6]
```diff
@@ -1,67 +1,59 @@
...
+
+### Install the plugin
+
+![Import the plugin](import-plugin.png)
+
+To install the plugin:
+
+1. In ReadyAPI, switch to the Integrations tab.
+2. Find ReadyAPI Postman Plugin and click Install.
+3. Confirm that you want to download and install the plugin.
```
Comment (IlyaAvdeev) [inline README.md]: need instructions how to install in SoapUI OS

**Comment by Alexander-Linkov** [comment]
As the plugin does not work with SoapUI OS, I leave the instructions only for ReadyAPI.

**Pull request merged**